docs/worker-node/install-singularity.md

```diff
@@ -19,14 +19,18 @@ keeps that privileged code to a
 vulnerability low.  Beginning with the kernel released with RHEL 7.4,
 there is a new
 [technology preview feature](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.4_Release_Notes/index.html#technology_previews_kernel)
-to allow unprivileged bind mounts in user name spaces, which allows
+to allow unprivileged bind mounts in user namespaces, which allows
 singularity to run as an unprivileged user.  The OSG has installed
 singularity in [cvmfs](install-cvmfs), so if you have a RHEL 7.4 kernel
 or later you
 can avoid installing singularity at all and reduce vulnerability even
 further.  The RHEL 7.4 kernel (version 3.10.0-693) is available as a
 security update for all RHEL 7 based versions, even on systems that
-have not updated to RHEL 7.4.
+have not updated to RHEL 7.4.  Enabling unprivileged user namespaces
```

Comment:
Can you pull this out as a separate admonition?  Something to be visually catching?